Source/ORTS.sln

```diff
@@ -55,110 +55,310 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ContentChecker", "ContentCh
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
-		Debug|Mixed Platforms = Debug|Mixed Platforms
-		Release|Mixed Platforms = Release|Mixed Platforms
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{9BE44F64-417D-4AA6-AF4D-2A6A0CBC814D}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{9BE44F64-417D-4AA6-AF4D-2A6A0CBC814D}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{9BE44F64-417D-4AA6-AF4D-2A6A0CBC814D}.Debug|Mixed Platforms.ActiveCfg = Debug|x86
 		{9BE44F64-417D-4AA6-AF4D-2A6A0CBC814D}.Debug|Mixed Platforms.Build.0 = Debug|x86
+		{9BE44F64-417D-4AA6-AF4D-2A6A0CBC814D}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{9BE44F64-417D-4AA6-AF4D-2A6A0CBC814D}.Debug|x64.Build.0 = Debug|Any CPU
```

Comment:
Not sure what's going on here, this looks like each project is going from just "Mixed Platforms" to having three configurations ("Mixed Platforms", "Any CPU", "x64"), which is surely wrong? We should only have "Any CPU" configurations, right?

Reply:
Seems to work with only "Any CPU" so expect you are right.